### ISSUE TITLE
impossible to install mb behind repository manager

### ISSUE THREAD

**Issue by francois-montmasson-efg** [bbyars/mountebank#141]
#### Expected behaviour

'npm install mountbank' does not work when using repository manager (e.g. Nexus, Artifactory).
#### Actual behaviour

Fail, because npm find shrinkwrap and try to find dependencies directly in npm registry in place of passing through configured registry. This is a know npm cli limitation...

c.f. https://support.sonatype.com/hc/en-us/articles/213465048-Why-does-npm-client-need-access-to-URLs-other-than-my-private-registry-
#### Steps to reproduce

First, install a repository manager. Nexus or Artifactory.
Then, configure the repository manager in order it act as a proxy to npmjs.org
configure npm in order to pass through your repository manager by setting config [registry](https://docs.npmjs.com/misc/registry)
Finally, try to npm install mountbank 
#### Software versions used

```
OS         : all
mountebank :  1.5.1
node.js    : 4.2.6
  (only if installed via npm)
Installation method : npm install
  (npm, zip, tar, pkg, deb, rpm)
```
#### Log contents in mb.log when running mb --loglevel debug

N/A

**Comment by francois-montmasson-efg** [comment]
Here is a possible solution: https://strongloop.com/strongblog/strong-pm-node-js-process-management-spotlight/
'sl-build -npm' seems most pragmatic solution.

**Comment by francois-montmasson-efg** [comment]
you can also use shonkwrap in order to produce a shrinkwrap that does not hard code repository.

`
{
  "name": "mountebank",
  "version": "1.5.1",
  "dependencies": {
    "accepts": {
      "version": "1.3.3"
    },
    "addressparser": {
      "version": "0.3.2"
    },
    "ansi-regex": {
      "version": "2.0.0"
    },
    "ansi-styles": {
      "version": "2.2.1"
    },
    "array-flatten": {
      "version": "1.1.1"
    },
    "asn1": {
      "version": "0.2.3"
    },
    "assert-plus": {
      "version": "0.2.0"
    },
`

**Comment by bbyars** [comment]
Fixed in 5edce0d
